netci.groovy

```diff
@@ -425,10 +425,10 @@ def targetGroupOsMap = ['netcoreapp': ['Windows 10', 'Windows 7', 'Windows_NT',
                 if (isPR) {
                     // Set PR trigger, we run Windows_NT, Ubuntu 14.04, CentOS 7.1, PortableLinux and OSX on every PR.
                     if ( osName == 'Windows_NT' || osName == 'Ubuntu14.04' || osName == 'CentOS7.1' || osName == 'OSX10.12' || osName== 'PortableLinux') {
-                        Utilities.addGithubPRTriggerForBranch(newJob, branch, "Innerloop ${osName} ${configurationGroup} ${archGroup} Build and Test")
+                        Utilities.addGithubPRTriggerForBranch(newJob, branch, "Innerloop ${targetGroup} ${osName} ${configurationGroup} ${archGroup} Build and Test")
```

Comment:
this will change existing names like
`Innerloop Windows_NT Debug x86 Build and Test `
to
`Innerloop netcoreapp Windows_NT Debug x86 Build and Test `
which is more to type.

Maybe only insert targetGroup into the name when it's != netcoreapp.?